src/sentry/quotas/base.py

```diff
@@ -217,6 +218,10 @@ class Quota(Service):
         "get_quotas",
         "get_blended_sample_rate",
         "get_transaction_sampling_tier_for_volume",
+        "assign_seat",
+        "unassign_seat",
+        "set_billing_seat_recreate",
```

Comment:
Feels suspicious that the word "billing" is in the sentry codebase

Reply:
Could call it `enable_seat_renewal` and `disable_seat_renewal` 🤷

Reply:
Does the phrase “renewal” have meaning in the context of sentry?

would “enable_seat_persistence” make more sense?